### DISCUSSION TITLE
e-pénéztárgép által kötelezően értelmezendő jelölések 4.6.1.

### DISCUSSION THREAD

**Discussion by PereszlenyiArpadECR-Trade** [nav-gov-hu/eRECEIPT#29] [Kérdések és válaszok]
Az 1.0-ás fejlesztői dokumentáció 4.6.1. pontjában lévő táblázat 9-es sorában a fizetőeszköz értékkészletnél nem szerepelnek az egyéb fizetőeszközök (10.6.1.20. pontban viszont fel vannak sorolva). Az ePTG-nek már nem kötelező használni az egyéb fizetőeszközöket, vagy csak kimaradt 4.6.1-nél. Vegyes fizetés esetén mi az ide vonatkozó szabályozás?

**Comment by BatariZ** [comment]
Az időközben megjelent rendelet (A nemzetgazdasági miniszter 8/2025. (III. 31.) NGM rendelete az e-pénztárgépek forgalmazásáról, üzemeltetéséről, valamint az e-pénztárgépek és az e-nyugta kiállításának követelményeiről), illetve az aktualizált Fejlesztői dokumentáció tartalmazza a válaszokat a feltett kérdésekre.
